kartograf/context.py

```diff
@@ -39,6 +37,12 @@ def __init__(self, args):
             self.epoch = self.args.epoch
             self.epoch_dir = "r" + self.args.epoch
             self.epoch_datetime = repro_epoch
+        # This is only for testing, to set an epoch in the Test context.
+        # the CLI parser prevents this argument from existing without the 'reproduce' arg
+        elif self.args.epoch:
```

Comment:
Hm, couldn't we do this from outside of the context by mocking the epoch? Unless it's much more complex I would prefer not to add test specific code in the normal code.

Reply:
yep, had to do a small refactor to make this work here [cfb044d](https://github.com/asmap/kartograf/pull/49/commits/cfb044d0f4a20448dcb45b206053d5bc0cebb568).